packages/edit-site/src/components/sidebar-navigation-screen/index.js

```diff
@@ -40,10 +40,15 @@ export default function SidebarNavigationScreen( {
 					) : (
 						<div className="edit-site-sidebar-navigation-screen__icon-placeholder" />
 					) }
-
-					<div className="edit-site-sidebar-navigation-screen__title">
-						{ title }
-					</div>
+					{ ! parentTitle ? (
+						<div className="edit-site-sidebar-navigation-screen__title">
```

Comment:
Should we just make this div an h2 to avoid the extra DOM element?

Reply:
Also since it's exactly the same element tree. you could avoid the ternary by doing `TagName = parentTitle ? 'div' : 'h2';` and `<TagName....`